addons/wp/stringtable.xml

```diff
@@ -375,7 +376,7 @@
         <Key ID="STR_Lambs_WP_Module_TaskPatrol_Range_Tooltip">
             <English>Max distance between waypoints</English>
             <German>Maximale Distanz zwischen zwei Wegpunkten.</German>
-            <Polish>Maksymalny zasięg pomiędzy punktami trasy</Polish>
+            <Polish>Maksymalny zasięg pomiędzy punktami nawigacyjnymi</Polish>

```

Comment:
```suggestion
            <Polish>Maksymalny dystans pomiędzy punktami nawigacyjnymi</Polish>
```

Wydaje mi się że "zasięg" nie pasuje w tym kontekście.

Reply:
Tak, dystans wydaje się lepszym rozwiązaniem, niedługo to poprawię.